server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CfLifecycle.java

```diff
@@ -0,0 +1,40 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+import java.io.IOException;
+
+import org.opentosca.toscana.core.transformation.TransformationContext;
+import org.opentosca.toscana.plugins.lifecycle.AbstractLifecycle;
+
+import org.slf4j.Logger;
+
+public class CfLifecycle extends AbstractLifecycle {
```

Comment:
Please change to CloudFoundryLifecycle. Someone could think this is the CloudFormationLifecycle.